algebird-core/src/main/scala/com/twitter/algebird/lsh/vector/VectorMath.scala

```diff
@@ -0,0 +1,87 @@
+package com.twitter.algebird.lsh.vector
+
+/**
+ * Vector math functions for LshVectors.
+ */
+object VectorMath {
```

Comment:
Seems like this is too much.  We only use dot here, and that could be put somewhere else (EuclideanHashFamily helper object, etc.) if we really need to add it.